python/pyarrow/plasma.pyx

```diff
@@ -206,11 +206,28 @@ cdef class PlasmaClient:
         c_string store_socket_name
         c_string manager_socket_name
 
-    def __cinit__(self):
+    def __cinit__(self, store_socket_name, manager_socket_name, int release_delay):
```

Comment:
As one rough edge in Cython, if you call the ctor with the wrong parameters you will generally segfault, which is why factory methods are usually safer. We can tackle this in a follow up patch